lib/rspec/shell/expectations/call_log.rb

```diff
@@ -38,7 +37,8 @@ def get_argument_count(*expected_argument_series, sub_command_series: [], positi
         end
 
         def called_with_no_args?
-          load_call_log_list.first["args"].nil?
+          call_log_list = load_call_log_list
+          !call_log_list.empty? and call_log_list.first["args"].nil?
```

Comment:
I think we may have had a bug here. If there was no call log file found, the rescue in the load_call_log_list method returned an [ ], which we would then be calling nil.first... on